modules/core/03-connection/types/msgs_test.go

```diff
@@ -178,39 +150,17 @@ func (suite *MsgTestSuite) TestNewMsgConnectionOpenTry() {
 }
 
 func (suite *MsgTestSuite) TestNewMsgConnectionOpenAck() {
-	clientState := ibctm.NewClientState(
-		chainID, ibctm.DefaultTrustLevel, ibctesting.TrustingPeriod, ibctesting.UnbondingPeriod, ibctesting.MaxClockDrift, clientHeight, commitmenttypes.GetSDKSpecs(), ibctesting.UpgradePath,
-	)
-
-	// Pack consensus state into any to test unpacking error
-	consState := ibctm.NewConsensusState(
-		time.Now(), commitmenttypes.NewMerkleRoot([]byte("root")), []byte("nextValsHash"),
-	)
-	invalidAny := clienttypes.MustPackConsensusState(consState)
-
-	// invalidClientState fails validateBasic
-	invalidClient := ibctm.NewClientState(
-		chainID, ibctm.DefaultTrustLevel, ibctesting.TrustingPeriod, ibctesting.UnbondingPeriod, ibctesting.MaxClockDrift, clienttypes.ZeroHeight(), commitmenttypes.GetSDKSpecs(), ibctesting.UpgradePath,
-	)
-	connectionID := "connection-0"
-
 	testCases := []struct {
 		name    string
 		msg     *types.MsgConnectionOpenAck
 		expPass bool
 	}{
-		{"invalid connection ID", types.NewMsgConnectionOpenAck("test/conn1", connectionID, clientState, suite.proof, suite.proof, suite.proof, clientHeight, clientHeight, ibctesting.ConnectionVersion, signer), false},
```

Comment:
hmm, don't think this test case should have been removed?

Reply:
This test case is still there. https://github.com/cosmos/ibc-go/pull/6937/files/1113d22d5defccfacbfa02d41f5f5466c97b2513#diff-a1e84e6050495cc94a24465e64e2e4254eec6bbe26951d6980eaff705c4ff4fcR160

Reply:
That's the "invalid counterparty connection ID" not "invalid connection ID"?

Reply:
I misclicked. Isn't that test case also there https://github.com/cosmos/ibc-go/pull/6937/files/1113d22d5defccfacbfa02d41f5f5466c97b2513#diff-a1e84e6050495cc94a24465e64e2e4254eec6bbe26951d6980eaff705c4ff4fcR159